egs/heroico/asr1/run.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
+
+# Copyright 2020 ARL (John Morgan)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=0
+stop_stage=100
+
+ngpu=2 # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=1      # verbose option
+resume=        # Resume the training from snapshot
+
+# feature configuration
+do_delta=false
+
+train_config=conf/train.yaml
+lm_config=conf/lm.yaml
+decode_config=conf/decode.yaml
+
+# rnnlm related
+lm_resume= # specify a snapshot file to resume LM training
+lmtag=             # tag for managing LMs
+
+# decoding parameter
+recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+n_average=10
+
+# data
+data=/mnt/corpora/LDC2006S37
+data_url=www.openslr.org/resources/39/LDC2006S37.tar.gz
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=train_sp
+train_dev=devtest
+recog_set="devtest test native nonnative"
+fbankdir=fbank
+lmdatadir=data/local/lm_train
+dict=data/lang_1char/${train_set}_units.txt
+
+
+if [ $stage -le -1 ] && [ $stop_stage -ge -1 ]; then
+    echo "$0 Stage -1: Data download."
+    wget $data_url
```

Comment:
Is it OK to download the LDC data?
Usually, we need to do some process (purchase or register) to obtain the LDC data.